mycore-ocfl/src/main/java/org/mycore/ocfl/layout/MCRStorageLayoutConfig.java

```diff
@@ -56,51 +52,16 @@ public boolean hasParameters() {
      * @param slotLayout MyCoRe Class SlotLayout, see MCRStore for more info
```

Comment:
I would probably remove the Class from here since its now generic.
```suggestion
     * @param slotLayout MyCoRe SlotLayout, see MCRStore for more info
```